### PR TITLE
(PDB-1934) remove two-database events query acceptance test

### PR DESCRIPTION
**Pull request by wkalt**
This is duplicated in the http facts test.

### CODE REVIEW

**Comment by wkalt** [comment]
this should address @senior 's concern.

File: test/puppetlabs/puppetdb/http/facts_test.clj [hunk 1]
```diff
@@ -579,57 +582,70 @@
...
+                   (assoc :database write-db-config))
+        read-db (-> read-db-config
                     defaulted-read-db-config
                     (init-db true))
-        write-db (-> (create-db-map)
+        write-db (-> write-db-config
                      defaulted-write-db-config
                      (init-db false))]
+
     (with-shutdown-after [read-db write-db]
```
Comment (wkalt) [inline test/puppetlabs/puppetdb/http/facts_test.clj]: this would be better as a macro

Comment (wkalt) [inline test/puppetlabs/puppetdb/http/facts_test.clj]: fixed

**Comment by ajroetker** [comment]
Lgtm @mullr you ok on this?

**Pull request merged**